packages/medusa/src/types/line-item.ts

```diff
@@ -0,0 +1,16 @@
+import { CalculationContextData } from "./totals"
+
+export type GenerateInputData = {
+  variantId: string
+  quantity: number
+}
+
+export type GenerateContext = {
```

Comment:
we could add a more descriptive type here. Maybe something like `GenerateLineItemContext`

Reply:
I though the name of the method would be enough, but I guess you are right ahah

Reply:
@carlos-r-l-rodrigues on another subject I have created a type `CalculationContextData` but I did not change the name of the props in purpose cause I don't want to integrate breaking changes like that in the scope of that pr. wdyt?